cmd/dstask.go

```diff
@@ -231,7 +231,7 @@ func main() {
 
 	case dstask.CMD_CONTEXT:
 		if len(os.Args) < 3 {
-			fmt.Printf("Current context: %s", context)
+			fmt.Printf("Current context: %s\n", context)
```

Comment:
Unrelated. 
Added a newline here. Calls of dstask context would result in 

``` bash
@ard0gg:~/dstask-develop/ard0gg$ dstask context
Current context: @ardogg:~/dstask-develop/ard0gg$
```

Now result in 
``` bash
@ard0gg:~/dstask-develop/ard0gg$ ./dstask context
Current context:
@ard0gg:~/dstask-develop/ard0gg$
```

Reply:
Oh, thanks. I never noticed that as I have a multiline prompt